examples/SpringTxFailoverExample/src/main/java/example/ExampleManagerImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example;
+
+import java.util.List;
+import java.util.Map;
+
+public class ExampleManagerImpl implements ExampleManager {
```

Comment:
The same suggestions I made on the `ExampleDaoImpl` can be made on this class too!

Reply:
Sorry i noticed this so late, but I think this class is unnecessary since it just passes through directly to the DAO class. Would be better to use the ExampleDAO class directly in the ExampleService.